src/ert/storage/local_experiment.py

```diff
@@ -130,6 +135,18 @@ def create(
             json.dumps(parameter_data, indent=2).encode("utf-8"),
         )
 
+        if ert_templates:
+            ert_templates_abs: list[tuple[str, str]] = []
+            for src, dst in ert_templates:
+                incoming_template_file_path = Path(src)
```

Comment:
This should be in a subfolder, if I have a template file called `parameter.json`, the error message is quite cryptic 😅 
```
Experiment failed with the following error: Expecting value: line 1 column 1 (char 0)
```

Reply:
Right. All templates should then go into `templates/` subfolder?